kolibri/plugins/device_management/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+
+  <KModal
+    :title="$tr('header')"
+    :submitText="$tr('submitButtonLabel')"
+    :cancelText="$tr('cancelButtonLabel')"
+    size="medium"
+    @submit="handleSubmit"
+    :submitDisabled="submitDisabled"
+    @cancel="$emit('cancel')"
+  >
+    <KButton
+      class="new-address-button"
+      :text="$tr('newAddressButtonLabel')"
+      @click="$emit('click_add_address')"
+      appearance="basic-link"
+      v-show="!newAddressButtonDisabled"
+    />
+
+    <UiAlert
+      v-if="uiAlertProps"
+      v-show="showUiAlerts"
+      :type="uiAlertProps.type"
+      :dismissible="false"
+    >
+      {{ uiAlertProps.text }}
+      <KButton
+        v-if="requestsFailed"
+        appearance="basic-link"
+        :text="$tr('refreshAddressesButtonLabel')"
+        @click="refreshAddressList"
+      />
+    </UiAlert>
+
+    <template v-for="(a, idx) in addresses">
+      <div :key="`div-${idx}`">
+        <KRadioButton
+          class="radio-button"
+          v-model="selectedAddressId"
+          :key="idx"
+          :value="a.id"
+          :label="a.device_name"
+          :description="a.base_url"
+          :disabled="!a.available || !a.hasContent"
+        />
+        <KButton
+          @click="removeAddress(a.id)"
```

Comment:
No extra confirmation of deletion :question: :fearful:

Reply:
@khangmach @jtamiace

Reply:
Jamie and I decided against the extra confirmation of deletion early on the design phase. Not a destructive action